src/main/java/Storage.java

```diff
@@ -0,0 +1,78 @@
+import java.io.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Date;
+
+public class Storage {
+    private String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> load() throws LunaBotException {
```

Comment:
minor nitpick - perhaps there could be comments in this method to guide the readers?